 Network gustavofbreunig_default  Creating
 Network gustavofbreunig_default  Created
 Container gustavofbreunig-db-1  Creating
 Container gustavofbreunig-db-1  Created
 Container api_1  Creating
 Container api_2  Creating
 Container api_2  Created
 Container api_1  Created
 Container nginx  Creating
 Container nginx  Created
Attaching to api_1, api_2, db-1, nginx
db-1   | The files belonging to this database system will be owned by user "postgres".
db-1   | This user must also own the server process.
db-1   | 
db-1   | The database cluster will be initialized with locale "en_US.utf8".
db-1   | The default database encoding has accordingly been set to "UTF8".
db-1   | The default text search configuration will be set to "english".
db-1   | 
db-1   | Data page checksums are disabled.
db-1   | 
db-1   | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1   | creating subdirectories ... ok
db-1   | selecting dynamic shared memory implementation ... posix
db-1   | selecting default max_connections ... 100
db-1   | selecting default shared_buffers ... 128MB
db-1   | selecting default time zone ... Etc/UTC
db-1   | creating configuration files ... ok
db-1   | running bootstrap script ... ok
nginx  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx  | /docker-entrypoint.sh: Configuration complete; ready for start up
db-1   | performing post-bootstrap initialization ... ok
db-1   | syncing data to disk ... ok
db-1   | 
db-1   | 
db-1   | Success. You can now start the database server using:
db-1   | 
db-1   |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1   | 
db-1   | initdb: warning: enabling "trust" authentication for local connections
db-1   | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1   | waiting for server to start.... done
db-1   | server started
db-1   | CREATE DATABASE
db-1   | 
db-1   | 
db-1   | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
db-1   | SET
db-1   | SET
db-1   | SET
db-1   | SET
db-1   | SET
db-1   | SET
db-1   | SET
db-1   | SET
db-1   | SET
db-1   | SET
db-1   | SET
db-1   | CREATE TABLE
db-1   | CREATE TABLE
db-1   | DO
db-1   | 
db-1   | 
db-1   | /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/postgresql.config_file
db-1   | 
db-1   | waiting for server to shut down.... done
db-1   | server stopped
db-1   | 
db-1   | PostgreSQL init process complete; ready for start up.
db-1   | 
nginx  | 2024/03/05 15:37:32 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.208.4:8080/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:37:32 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.208.3:8080/clientes/1/extrato", host: "localhost:9999"
api_2  | 
api_2  |   .   ____          _            __ _ _
api_2  |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
api_2  | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
api_2  |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
api_2  |   '  |____| .__|_| |_|_| |_\__, | / / / /
api_2  |  =========|_|==============|___/=/_/_/_/
api_2  |  :: Spring Boot ::                (v3.2.2)
api_2  | 
api_1  | 
api_1  |   .   ____          _            __ _ _
api_1  |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
api_1  | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
api_1  |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
api_1  |   '  |____| .__|_| |_|_| |_\__, | / / / /
api_1  |  =========|_|==============|___/=/_/_/_/
api_1  |  :: Spring Boot ::                (v3.2.2)
api_1  | 
api_2  | 2024-03-05T15:37:34.724Z  INFO 1 --- [           main] c.r.r.RinhadrivedesignApplication        : Starting RinhadrivedesignApplication v1.0.0 using Java 17.0.10 with PID 1 (/app.jar started by root in /)
api_2  | 2024-03-05T15:37:34.728Z  INFO 1 --- [           main] c.r.r.RinhadrivedesignApplication        : No active profile set, falling back to 1 default profile: "default"
api_1  | 2024-03-05T15:37:34.821Z  INFO 1 --- [           main] c.r.r.RinhadrivedesignApplication        : Starting RinhadrivedesignApplication v1.0.0 using Java 17.0.10 with PID 1 (/app.jar started by root in /)
api_1  | 2024-03-05T15:37:34.825Z  INFO 1 --- [           main] c.r.r.RinhadrivedesignApplication        : No active profile set, falling back to 1 default profile: "default"
nginx  | 2024/03/05 15:37:34 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:37:36 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:37:38 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api_2  | 2024-03-05T15:37:40.324Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
api_1  | 2024-03-05T15:37:40.519Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
api_2  | 2024-03-05T15:37:40.718Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 381 ms. Found 2 JPA repository interfaces.
api_1  | 2024-03-05T15:37:40.838Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 308 ms. Found 2 JPA repository interfaces.
nginx  | 2024/03/05 15:37:40 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:37:42 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:37:44 [error] 29#29: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.208.3:8080/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:37:44 [error] 29#29: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.208.4:8080/clientes/1/extrato", host: "localhost:9999"
api_2  | 2024-03-05T15:37:45.116Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port 8080 (http)
api_2  | 2024-03-05T15:37:45.130Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
api_2  | 2024-03-05T15:37:45.130Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.18]
api_1  | 2024-03-05T15:37:45.232Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port 8080 (http)
api_1  | 2024-03-05T15:37:45.244Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
api_1  | 2024-03-05T15:37:45.315Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.18]
api_2  | 2024-03-05T15:37:45.736Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
api_2  | 2024-03-05T15:37:45.737Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 10617 ms
api_1  | 2024-03-05T15:37:45.841Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
api_1  | 2024-03-05T15:37:45.917Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 10718 ms
nginx  | 2024/03/05 15:37:46 [error] 29#29: *12 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api_2  | 2024-03-05T15:37:48.345Z  INFO 1 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
api_1  | 2024-03-05T15:37:48.521Z  INFO 1 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
api_2  | 2024-03-05T15:37:48.717Z  INFO 1 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.4.1.Final
api_1  | 2024-03-05T15:37:48.824Z  INFO 1 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.4.1.Final
api_2  | 2024-03-05T15:37:48.832Z  INFO 1 --- [           main] o.h.c.internal.RegionFactoryInitiator    : HHH000026: Second-level cache disabled
nginx  | 2024/03/05 15:37:48 [error] 29#29: *13 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api_1  | 2024-03-05T15:37:49.020Z  INFO 1 --- [           main] o.h.c.internal.RegionFactoryInitiator    : HHH000026: Second-level cache disabled
api_2  | 2024-03-05T15:37:50.036Z  INFO 1 --- [           main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
api_1  | 2024-03-05T15:37:50.219Z  INFO 1 --- [           main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
api_2  | 2024-03-05T15:37:50.220Z  WARN 1 --- [           main] com.zaxxer.hikari.HikariConfig           : HikariPool-1 - maxLifetime is less than 30000ms, setting to default 1800000ms.
api_2  | 2024-03-05T15:37:50.220Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
api_1  | 2024-03-05T15:37:50.333Z  WARN 1 --- [           main] com.zaxxer.hikari.HikariConfig           : HikariPool-1 - maxLifetime is less than 30000ms, setting to default 1800000ms.
api_1  | 2024-03-05T15:37:50.333Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
nginx  | 2024/03/05 15:37:50 [error] 29#29: *14 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api_2  | 2024-03-05T15:37:51.235Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@722b3ba2
api_2  | 2024-03-05T15:37:51.237Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
api_1  | 2024-03-05T15:37:51.317Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@1d2d4d7a
api_1  | 2024-03-05T15:37:51.319Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
api_2  | 2024-03-05T15:37:51.428Z  WARN 1 --- [           main] org.hibernate.orm.deprecation            : HHH90000025: PostgreSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)
api_1  | 2024-03-05T15:37:51.436Z  WARN 1 --- [           main] org.hibernate.orm.deprecation            : HHH90000025: PostgreSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)
nginx  | 2024/03/05 15:37:52 [error] 29#29: *15 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:37:54 [error] 29#29: *16 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
db-1 exited with code 137
api_2 exited with code 137
api_1 exited with code 137
nginx  | 2024/03/05 15:38:31 [error] 29#29: *17 connect() failed (113: No route to host) while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.208.3:8080/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:38:49 [error] 29#29: *17 connect() failed (113: No route to host) while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.208.4:8080/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:38:54 [error] 29#29: *20 connect() failed (113: No route to host) while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.208.3:8080/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:38:54 [error] 29#29: *20 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:38:56 [error] 29#29: *22 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:38:58 [error] 29#29: *23 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:39:04 [error] 29#29: *24 connect() failed (113: No route to host) while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.208.4:8080/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:39:04 [error] 29#29: *24 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:39:09 [error] 29#29: *26 connect() failed (113: No route to host) while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.208.3:8080/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:39:09 [error] 29#29: *26 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:39:11 [error] 29#29: *28 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/03/05 15:39:13 [error] 29#29: *29 no live upstreams while connecting to upstream, client: 192.168.208.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx exited with code 0
